src/Simplex/Chat/Types.hs

```diff
@@ -123,6 +123,9 @@ directContact :: Contact -> Bool
 directContact Contact {contactUsed, activeConn = Connection {connLevel, viaGroupLink}} =
   (connLevel == 0 && not viaGroupLink) || contactUsed
 
+anyDirectContact :: Contact -> Bool
```

Comment:
why `directContact` doesn't suffice? I see the the difference in what they're checking, but don't understand the purpose, maybe name is not descriptive enough

Reply:
because we should not be prohibiting the direct messages to group admins, as otherwise they cannot even send the group invitation. We might special-case it and say that direct messages with this admin are only allowed until there is a group, but I don't see much damage to allow messaging the user whose link you used to join.

Reply:
at least on backend level.

This case was actually clear only after failing group links test, I didn't get it initially.